pkg/shell/machine-dialogs.js

```diff
@@ -417,6 +417,9 @@ define([
 
             dialog.run(try_to_connect(dialog.address), function (ex) {
                 if (ex.problem == "no-host") {
+                    if (dialog.address.indexOf(":") == -1)
+                        dialog.address += ":22";
+                    ex.message = "Cockpit could not contact the given host "+dialog.address+". Make sure it has ssh running on port 22, or specify another port in the address.";
```

Comment:
String needs to be marked as translatable and use cockpit.format for string interpolation.